### ISSUE TITLE
Stronger End Game Stuff?

### ISSUE THREAD

**Issue by pbtenchi** [Azanor/thaumcraft-suggestions#97]
I find thaumcraft is often over shadowed by other mods, with a slower progression and worse endgame. The only thing REALLY worth while is the warding ability. Perhaps pocket dimensions like in AE2, or the ability to add runic shielding to other armors. It would also be nice to not have to rely so much on soap to deal with warp, perhaps a bauble to deal with warp?

**Comment by carso1500** [comment]
yes, thaumcraft compared to other mods is really difficult to obtain end game stuff and the things you obtain aren´t as rewarding as you expect to be, more powerfull stuff and some balance would be gratefull
